src/systrack/arch/__init__.py

```diff
@@ -43,6 +45,8 @@
 	'riscv'        : lambda v: ArchRiscV(v, abi='rv32', bits32=True),
 	'riscv64'      : lambda v: ArchRiscV(v, abi='rv64'),
 	'riscv64-32'   : lambda v: ArchRiscV(v, abi='rv32'),
+	's390x'        : lambda v: ArchS390(v, abi='64'),
+	's390'         : lambda v: ArchS390(v, abi='32'),
```

Comment:
```suggestion
	's390x'        : lambda v: ArchS390(v, abi='s390x'), # "64" ABI
	's390'         : lambda v: ArchS390(v, abi='s390'),  # "32" ABI
```

I see you are documenting the ABIs in the README table as "s390x" and "s390", but then use `64` and `32` for `--arch help` output and for the `abi=` here. Maybe it's better to use `'s390x'`/`'s390'` everywhere at this point. Considering that the corresponding ELF ABI also uses "s390x" and "s390" that seems reasonable.

Don't mind the fact that we try keeping the `abi=` in line with what the `syscall.tbl` file contains, in some cases it might make sense to use different names.

Same goes for all the other occurrences like in `--arch help` output, `ArchS390` class etc.

Reply:
Ok.